src/css/site.styl

```diff
@@ -194,11 +196,35 @@ input, select
 
 // ******************************************************
 
+// BLOG
+
+#about-blog, #about-blog-single
+  iframe
+    max-width: 100%
+
+#blog-border-block
+  display: none
+  z-index: 300
+
+.blog-item-mobile
+  max-width: 400px
+
+h2.blog-title
```

Comment:
does this need to be tag qualified? will clash if in the future the markup wants to change